From 50be77457eca6f72b62647943f16d9b9cdd5f3f1 Mon Sep 17 00:00:00 2001
From: Guan Wen <guanx.wen@intel.com>
Date: Fri, 12 Jun 2015 11:23:28 +0800
Subject: [PATCH 427/462] ov8858: remove 1080p video resolution

Preview fov not correct when switch from camera mode to video mode in
1080p resolution, due to current sensor 1080p setting is cropped from 8M.

Remove the 1080p video resolution.

Change-Id: I826a9cd405e9732eb75f25647543c1cbb3e8b628
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-39241
Signed-off-by: Guan Wen <guanx.wen@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/60128
Reviewed-by: DaleiX Liu <daleix.liu@intel.com>
Build: IMIN (SC)
Approver: Jing Tao <jing.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/i2c/ov8858.h              | 38 ----------------------
 1 file changed, 38 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
index c161031..c9f02b8 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
@@ -1343,44 +1343,6 @@ static struct ov8858_resolution ov8858_res_video[] = {
 		},
 	},
 	{
-		.desc = "ov8858_1080P_30_VIDEO",
-		.width = 1936,
-		.height = 1096,
-		.used = 0,
-		.regs = ov8858_1080P_30,
-		.bin_factor_x = 0,
-		.bin_factor_y = 0,
-		.skip_frames = 1,
-		.fps_options =  {
-			{
-				.fps = 30,
-				.pixels_per_line = 3880,
-				.lines_per_frame = 2573,
-			},
-			{
-			}
-		},
-	},
-	{
-		.desc = "ov8858_1080P_60_VIDEO",
-		.width = 1936,
-		.height = 1096,
-		.used = 0,
-		.regs = ov8858_1080P_60,
-		.bin_factor_x = 0,
-		.bin_factor_y = 0,
-		.skip_frames = 1,
-		.fps_options =  {
-			{
-				.fps = 60,
-				.pixels_per_line = 3880,
-				.lines_per_frame = 1260,
-			},
-			{
-			}
-		},
-	},
-	{
 		.desc = "ov8858_5M_VIDEO",
 		.width = 2592,
 		.height = 1466,
-- 
1.9.1

